docs/admin/deployment.rst

```diff
@@ -291,9 +291,9 @@ you create:
 
 ``CSRF_TRUSTED_ORIGINS``
    Optional. A list of trusted origins for unsafe requests. It should contain the domains
```

Comment:
Splitting up this suggestion because GitHub can't deal with it at once.

We should match the other list descriptions.
```suggestion
   Optional. A comma-separated list of trusted origins for unsafe requests.
```